Cargo.toml

```diff
@@ -16,4 +16,4 @@ bit_field = "0.10.0"
 riscv-target = "0.1.2"
 
 [features]
-inline-asm = []
+inline-asm = []
```

Comment:
why is this required?

Reply:
I think that is an option in some of the macros